lib/vsc/administration/slurm/sync.py

```diff
@@ -116,11 +116,15 @@ def slurm_project_qos(projects, slurm_qos_info, clusters, protected_qos, qos_cle
             qos_name = f"{cluster}-{project.name}"
             if qos_name not in cluster_qos_names:
                 commands.append(create_add_qos_command(qos_name))
-            commands.append(create_modify_qos_command(qos_name, {
-                "GRPTRESMins": "billing={credits},gres/gpu={gpuminutes}".format(
-                    credits=60*int(project.credits),
-                    gpuminutes=max(1, 60*int(project.gpu_hours)))
-                }))
+
+            try: 
+                commands.append(create_modify_qos_command(qos_name, {
+                    "GRPTRESMins": "billing={credits},gres/gpu={gpuminutes}".format(
```

Comment:
missing an `f` before the string

Reply:
pylint does not always nag about `format`, it might be the named formatting it says should just be a fstring.

Reply:
ok, will change